src/js/components/account/login.js

```diff
@@ -35,7 +39,15 @@ export class Login {
 
     fields.forEach((field) => {
       const $formElementsContainer = $("<div />");
-      $formElementsContainer.append(getLabel(field.label));
+      $formElementsContainer.append(
+        getLabel($webflowForm, {
+          htmlFor: `my-muni-${field.label
+            .split(" ")
+            .join("_")
+            .toLocaleLowerCase()}`,
+          text: field.label,
+        })
+      );
```

Comment:
Why are we slugifying again? We need to be precise and hardcode field names instead of slugifying them from labels

Reply:
field names are for computers. Labels are for humans. I'm very uncomfortable with making an assumption that we can reliably turn a label into a field name. Even if this never breaks, it sets a dangerous precedent. The corrct approach here is to include the label in the list of objects above.

Reply:
Agreed, I did not want to mess too much with this that work right now so, I made changes where I felt comfortable and left the rest for refactoring later.